crates/aranya-afc-util/src/handler.rs

```diff
@@ -149,12 +149,12 @@ pub struct BidiChannelCreated<'a> {
 pub struct BidiChannelReceived<'a> {
     /// The unique ID of the previous command.
     pub parent_cmd_id: Id,
-    /// The channel author's user ID.
-    pub author_id: UserId,
+    /// The channel author's Device ID.
```

Comment:
nit: `Device` as a standalone word (here and elsewhere) probably doesn't need to be capitalized.